homeassistant/components/notify/html5.py

```diff
@@ -334,12 +361,28 @@ def targets(self):
             targets[registration] = registration
         return targets
 
+    def dismiss(self, **kwargs):
+        """Dismisses a notification."""
+        data = kwargs.get(ATTR_DATA)
+        tag = data.get(ATTR_TAG) if data else ""
+        payload = {
+            ATTR_TAG: tag,
+            ATTR_DISMISS: True,
+            ATTR_DATA: {}
+        }
+
+        self._push_message(payload, **kwargs)
+
+    async def async_dismiss(self, **kwargs):
```

Comment:
This seems not needed. We can just use `dismiss` above and use that as target for the service handler.